### PR TITLE
ENT - RMP-206 "highlighted text of quick search dorpdown when the substring is matching"

### PR DESCRIPTION
**Pull request by gayan-orangehrm**

### CODE REVIEW

File: components/src/core/components/List/List.vue [hunk 1]
```diff
@@ -88,7 +88,9 @@
             </template>
             <template v-slot:option="{data}">
               <oxd-profile-pic size="extra-small" :imageSrc="data.avatar_url" />
-              <span class="margin-left">{{ data.label }}</span>
+              <div class="margin-left">
+                <div v-html="highlight(data.label)"></div>
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/List/List.vue:92]: https://stackoverflow.com/a/66023390/5748470

[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/List/List.vue]: @lgtkaushalya @Super-Chama sanitized v-html in the input using [sanitize-html node module](https://www.npmjs.com/package/sanitize-html)

File: components/src/core/components/List/List.vue [hunk 2]
```diff
@@ -88,7 +88,9 @@
             </template>
             <template v-slot:option="{data}">
```
Comment (Super-Chama) [inline components/src/core/components/List/List.vue:89]: As we discussed use the highlighted text passed from option slot of autocomplete. and add the sanitiser at autocomplete.
https://github.com/orangehrm/oxd/blob/ent/components/src/core/components/Input/Autocomplete/AutocompleteInput.vue#L47

Comment (gayan-orangehrm) [reply components/src/core/components/List/List.vue]: Updated as per discussion

**Review by Super-Chama — APPROVED** [review]

**Pull request merged**